Units/parser-cxx.r/class-inheritance.cpp.d/expected.tags

```diff
@@ -0,0 +1,10 @@
+A	input.cpp	/^class A$/
+B	input.cpp	/^class B$/
+C	input.cpp	/^template<typename X,typename Y> class C$/
+D	input.cpp	/^class D : public A$/;"	inherits:A
+E	input.cpp	/^class E : public A, public B$/;"	inherits:A,B
+F	input.cpp	/^class F : private A, public B$/;"	inherits:A,B
+G	input.cpp	/^class G : virtual D$/;"	inherits:D
+H	input.cpp	/^class H : public A, virtual B$/;"	inherits:A,B
+I	input.cpp	/^class I : public C<A,B>$/;"	inherits:C<A,B>
+J	input.cpp	/^class J : private A, public C<A,B>$/;"	inherits:A,C<A,B>
```

Comment:
You're right, the old parser did strip template information from base classes.

As a policy I'd like to provide as more information as possible. I'm holding also another change that will add template information to classes in the following form:

template<typename A,typename B> class C : public D<A,B>

C   file.cpp   /whatever/    class     inherits:D<A,B>    template:A,B

So one knows that A and B are template placeholders and can actually substitute real types in place of that (instead of trying to look up A and B somewhere elsee).

Yet more.. it might be nice to even have a

  inherits: public D<A,B>

since the access type of inheritance defines the default access level of the inherited members.

However, all of this is incompatible.

One idea would be to have C/C++ parser specific switches that enable incompatible behaviours.
Something like: --c++-options=+ta. If no option is passed the template and access information is stripped (backward compatibility). If +t is passed then template information is preserved. If +a is passed then access information is preserved.

But all of this requires parser specific options...